ui/src/message_popup/author/question_page.jsx

```diff
@@ -0,0 +1,284 @@
+/* @flow weak */
+/* eslint no-console: "off" */
+import React from 'react';
+import _ from 'lodash';
+import VelocityTransitionGroup from 'velocity-react/velocity-transition-group';
+import 'velocity-animate/velocity.ui';
+
+import * as EditingComponents from './question_editing_components/editing_component.jsx';
+
+import {allQuestions} from '../questions.js';
+import {allStudents} from '../../data/virtual_school.js';
+import {learningObjectives} from '../../data/learning_objectives.js';
+import {indicators} from '../../data/indicators.js';
+import * as Routes from '../../routes.js';
+
+import AppBar from 'material-ui/AppBar';
+import IconButton from 'material-ui/IconButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+
+import ArrowBackIcon from 'material-ui/svg-icons/navigation/arrow-back';
+
+export default React.createClass({
+  displayName: 'QuestionPage',
+
+  propTypes: {
+    originalQuestion: React.PropTypes.object,
```

Comment:
Would it make sense to just use `originalQuestion` and remove the other props here?  That seems like the essential difference between `new` and `edit`, and then `getInitialState` could just read from the original question if there was one or fall back to the default.

That would also allow the code to form `goodExamplesText` and `badExamplesText` to be moved here (where it seems like it really belongs, since the newline-delimited format is specific to this UI).